test/utils/mocks/body.html

```diff
@@ -9,6 +9,9 @@
     <!-- Picture Decorating -->
     <p><em><picture><img src="/test/mocks/media_.png"/></picture></em></p>
     <p><strong><picture><img src="/test/mocks/media_.png"/></picture></strong></p>
+    <!-- Picture Link Decorating -->
+    <p><picture><img src="/test/mocks/media_.png" alt="https://www.adobe.com#_blank | Image description"/></picture></p>
```

Comment:
There should be specific unit-tests for checking image links.

Reply:
👍 I added these unit tests and moved them into their own test files.